xbmc/music/Album.cpp

```diff
@@ -181,8 +181,8 @@ const std::vector<std::string> CAlbum::GetAlbumArtist() const
   //When artist credits have not been populated attempt to build an artist vector from the descrpition string
   //This is a tempory fix, in the longer term other areas should query the album_artist table and populate
   //artist credits. Note that splitting the string may not give the same artists as held in the album_artist table
-  if (albumartists.empty() && !strArtistDesc.empty())
-    albumartists = StringUtils::Split(strArtistDesc, g_advancedSettings.m_musicItemSeparator);
+//  if (albumartists.empty() && !strArtistDesc.empty())
+//    albumartists = StringUtils::Split(strArtistDesc, g_advancedSettings.m_musicItemSeparator);
```

Comment:
I introduced those lines as a temp fix a week ago, as the comment above them says. Here I am trying with them out. I think I have fixed the issue, but I want to get some real data testing going to be absolutely sure. Don't want to make the same mistake twice!